torchvision/models/resnet.py

```diff
@@ -194,7 +194,8 @@ def _make_layer(self, block, planes, blocks, stride=1, dilate=False):
 
         return nn.Sequential(*layers)
 
-    def forward(self, x):
+    # Allow for accessing forward method in a inherited class
+    def _forward(self, x):
```

Comment:
Same comment here, can we invert and use instead `def forward` and then in the other line `_forward = forward`?

Reply:
Tried that, problem is that if we define forward and then assign _forward = forward in float, then the quantized model still needs a forward, which clashes with the definition of forward in the parent model, making it not work.